pkg/nuclide/ocaml-base/lib/MerlinProcess.js

```diff
@@ -269,6 +270,18 @@ function getPathToMerlin(): string {
     && global.atom.config.get('nuclide.nuclide-ocaml.pathToMerlin') || 'ocamlmerlin';
 }
 
+/**
+ * @return The set of arguments to pass to ocamlmerlin.
+ */
+function getMerlinFlags(): Array<String> {
+  const configVal = global.atom
+    && global.atom.config.get('nuclide.nuclide-ocaml.merlinFlags');
+  // To split while stripping out any leading/trailing space, we match on all
+  // *non*-whitespace.
+  const configItems = configVal && configVal.match(/\S+/g);
```

Comment:
I looked into that library and its probably overkill for merlin's use case (since in practice there's only one/two command line flag that you'd actually use and the rest should appear project scoped to a `.merlin` library).
I'm happy to add it if you feel strongly.